service/src/main/kotlin/app/cash/backfila/service/SlackHelper.kt

```diff
@@ -52,7 +51,11 @@ class SlackHelper @Inject constructor(
   }
 
   private fun nameAndId(run: DbBackfillRun) =
-    "[${env.name}] ${run.service.registry_name} `${run.registered_backfill.name}` (${idLink(run.id)})"
+    "[${deployment.name}] ${run.service.registry_name} `${run.registered_backfill.name}` (${
```

Comment:
Let's not do this.

Reply:
undo this.

Reply:
Env/Environment is gone, what do you want?

Reply:
maybe he meant the formatting? I would bring it back to one line personally, or if its too long use a + before the last space

Reply:
Yeah the formatting change that adds newlines.